### ISSUE TITLE
Margins resizers doesn't work correctly for first folder note if it's Super Note

### ISSUE THREAD

**Issue by Intreeguant** [standardnotes/forum#2401]
**Describe the bug**
If adjustable margins are enabled, when navigating between folders, there is a problem displaying the note if it is a Super Note.

The set margins are not applied and the note appears as if compressed onto a small part of the screen.
Simply select another note in the same folder and then return to the previous note for the display problem to disappear.

Selecting another folder, then returning to the previous folder, the display bug returns.

**To Reproduce**
1. Enable Margin Resizers in Settings
2. Create two notes in a folder: one Super note which has to be the first one in view, and another one (plain text for example)
3. Resize Super note margins on left and right
4. Create another folder with empty note
5. Return back to the first folder and Super Note shoud be squezed
6. Select the second note and go back to Super note: margins resized worked again
7. Click on the second folder and return back to the first one
8. Super note view is squeezed again

**Expected behavior**
Resized margins option should work for first Super Note displayed in folder even when switching between folders.

**Desktop (please complete the following information):**
 - OS: Linux Fedora 36
 - Browser: Firefox 108.0.1

**Comment by effieeee** [comment]
Hi @Intreeguant, I'm afraid I was unable to reproduce this with the steps you've provided, though I wanted to clarify if you also use the app fully maximized, or do you have it in a windowed view? Does the issue also persist even after disabling and reenabling the margin resizers, or signing out and signing back into your account on the app?

**Comment by Intreeguant** [comment]
Hi @effieeee, thanks for your reply. The problem occured when I use maximized or windowed view. I disabled and reenabled margin resizers, log out and sign in back into my account and nothing worked. But I found only some folders were affected by this problem and it seems I created them before enabling margins resizers for the first time. So I deleted them, created new folders and move my notes into them and it solved my problem. I don't have any view problem now.
